modules/evm/rpc/src/lib.rs

```diff
@@ -128,9 +128,9 @@ fn to_u128(val: NumberOrHex) -> std::result::Result<u128, ()> {
 }
 
 // 20M. TODO: use value from runtime
```

Comment:
```suggestion
// 33M. TODO: use value from runtime
```

Reply:
@xlc still working. I'm thinking about moving the limit to runtime